ccd-definition/CaseEvent.json

```diff
@@ -29,5 +29,21 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Confirm service"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "REQUEST_EXTENSION",
+    "Name": "Request extension",
+    "Description": "Defendant solicitor is submitting requesting for an extension of time",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "*",
```

Comment:
is it true to say this is available from any states? When 336 introduces STAYED state it will mean this event is triggerable

Reply:
we can pass in multiple states here so we can add states as we develop :)

Reply:
done